distribution/containers/Dockerfile.distroless

```diff
@@ -42,5 +49,15 @@ FROM flood as rtorrent-flood
 # Install rTorrent
 COPY --from=rtorrent / /
 
+# Install tini
+COPY --from=build /sbin/tini /sbin/tini
+
+ENV \
+    HOME=/home/download \
+    FLOOD_OPTION_HOST="0.0.0.0" \
+    FLOOD_OPTION_PORT="3000" \
+    FLOOD_OPTION_RTORRENT="true"
+
 # Flood with managed rTorrent daemon
-ENTRYPOINT ["flood", "--host=0.0.0.0", "--rtorrent"]
+ENTRYPOINT ["/sbin/tini", "--"]
+CMD ["flood"]
```

Comment:
we still probably want to use `flood` as the entrypoint.

Reply:
Updated